examples/images_contours_and_fields/interpolation_none_vs_nearest.py

```diff
@@ -1,62 +0,0 @@
-"""
```

Comment:
I've added some detailed text to interpolation methods, mostly copy pasting from the example I removed.

I am stressing once again that we need to cut back the number of examples. Most are extremely poorly designed, with unclear message and can be merged into one single example. It drowns the information too much.

To be specific, we have more than 500 examples right now, many displaying several images. There is no way anyone can sort or process any kind of information from these. If it helps convince you, I can have a communication expert look at the examples and the text and help us with the process of cleaning up our gallery.